packages/data/addon/adapters/facts/elide.ts

```diff
@@ -78,6 +78,10 @@ export default class ElideFactsAdapter extends EmberObject implements NaviFactAd
         return `${fieldStr}=ge=(${filterVals[0]});${fieldStr}=le=(${filterVals[1]})`;
       }
 
+      if (operator === 'nbet') {
+        return `${fieldStr}=le=(${filterVals[0]});${fieldStr}=ge=(${filterVals[1]})`;
```

Comment:
Should this be:
```suggestion
        return `${fieldStr}=lt=(${filterVals[0]}),${fieldStr}=gt=(${filterVals[1]})`;
```

Reply:
Don't we want an `OR`?

Reply:
Yup you're correct, would that just be || in this case?

Reply:
The suggestion above is the correct syntax ^^